src/System.Diagnostics.Tracing/src/System/Diagnostics/Tracing/EventCounter.cs

```diff
@@ -264,7 +286,8 @@ private void OnEventSourceCommand(object sender, EventCommandEventArgs e)
                 float value;
                 if (e.Arguments.TryGetValue("EventCounterIntervalSec", out valueStr) && float.TryParse(valueStr, out value))
                 {
-                    EnableTimer(value);
+                    lock (this)      // Lock the EventCounterGroup
+                        EnableTimer(value);
```

Comment:
> my inclination right now is to do nothing (effectively make it the EventListener's issue)

Ok.  Might at least be worth adding a comment about; in case that unlikely situation does arise, it might at least help shed some light on the issue.